src/murmur/Meta.h

```diff
@@ -125,6 +129,9 @@ class MetaParams {
 	QVariant qvSuggestPositional;
 	QVariant qvSuggestPushToTalk;
 
+	/// qsAbsSettingsFn is the absolute path to
+	/// the murmur.ini used by this Meta instance.
+	QString qsAbsSettingsFn;
```

Comment:
It’s a file path, not a file name tho, right? :)

The commit description still talks about `qAbsSettingsFn`